addons/common/functions/fnc_initDisplayMultiplayerSetup.sqf

```diff
@@ -4,6 +4,9 @@
 
 params ["_display"];
 
+// Deprecated in newer versions, replaced by CBA system
+// https://github.com/CBATeam/CBA_A3/wiki/Name-Groups-in-Lobby
+if ([1,0,1] isEqualTo getArray (missionConfigFile >> "tmf_version")) then {
```

Comment:
Swap to dedicated `[1,1,0] call TMF_common_fnc_checkTMFVersion`, this does a nice comparision